.gitmodules

```diff
@@ -4,3 +4,6 @@
 [submodule "lib/openimu-core"]
 	path = lib/openimu-core
 	url = https://github.com/tidewise/openimu-core
```

Comment:
If you want to use submodules, I suggest you remove this one that we don't use nor need.